packages/react/src/drafts/MarkdownEditor/_ToolbarButton.tsx

```diff
@@ -7,7 +7,9 @@ export const ToolbarButton = forwardRef<HTMLButtonElement, IconButtonProps>((pro
   const {disabled, condensed} = useContext(MarkdownEditorContext)
 
   return (
+    // eslint-disable-next-line primer-react/a11y-remove-disable-tooltip
     <IconButton
+      unsafeDisableTooltip={true}
```

Comment:
this component is not maintained anymore and the tests are dramatically failing when the icon button is enabled. So I simply disabled. Let me know if you have any concern 🙌🏻